go/pkg/client/cas.go

```diff
@@ -1372,7 +1372,8 @@ func (c *Client) downloadNonUnified(ctx context.Context, execRoot string, output
 				if err != nil {
 					return err
 				}
-				for dg, data := range bchMap {
+				for _, dg := range batch {
```

Comment:
Wait, did I accidentally break this functionality when refactoring for the unified CAS ops? If I did, we need the same change in downloadBatch, too, right?